tests/LocaleTest.php

```diff
@@ -91,31 +91,36 @@ public function testInvalidLocalesFormat(array $supportedLocales): void
         $this->createMiddleware($supportedLocales);
     }
 
-    public function dataLocaleFromPath(): array
+    public function testLocaleFromPathMatchesDefaultLocale(): void
     {
-        return [
-            'matches default locale' => ['en', 'en-US', 'en'],
-            'does not match default locale' => ['uz', 'uz-UZ', 'uz'],
-        ];
+        $request = $this->createRequest('/en/home?test=1');
+        $middleware = $this->createMiddleware(
+            locales: ['en' => 'en-US', 'uz' => 'uz-UZ'],
+            cookieDuration: new DateInterval('P5D'),
+        );
+
+        $response = $this->process($middleware, $request);
+
+        $cookies = CookieCollection::fromResponse($response)->toArray();
+
+        $this->assertSame(Status::FOUND, $response->getStatusCode());
+        $this->assertSame('/home?test=1', $response->getHeaderLine(Header::LOCATION));
+        $this->assertArrayHasKey('_language', $cookies);
```

Comment:
It's already tested in `testSaveLocaleWithCustomArguments`.

Reply:
Here need check that cookies is send.